workos/src/main/kotlin/com/workos/WorkOS.kt

```diff
@@ -164,12 +211,7 @@ class WorkOS(
       }
       else -> {
         val responseData = mapper.readValue(response.body(), GenericErrorResponse::class.java)
-
-        if (responseData.error != null || responseData.error_description != null) {
-          throw OauthException(responseData.message, status, requestId, responseData.error, responseData.error_description)
```

Comment:
Deleted this -- we do not need it now that we are enforcing an API key in the constructor.